package swm.virtuoso.reviewservice.exception

class EmptyPathException : RuntimeException("비어있는 경로가 포함되어 있습니다.")